tools/qcxms/qcxms_getres.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="qcxms_getres" name="QCxMS get results" version="@TOOL_VERSION@+galaxy0" profile="21.05">
+    <description>Get result of the Production run to obtain a QCxMS simulated mass spectrum</description>
+    
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="edam"/>
+    <expand macro="creator"/>
+    <expand macro="requirements"/>
```

Comment:
Do we just have a single container that has all the executables? Technically we could use the bioconda package here as this only requires the plotms package?

Reply:
So apparently conda version is producing different output in each run while the container version produces the same output.

Reply:
switched back to docker